ros_gz_sim/launch/ros_gz_spawn_model.launch.py

```diff
@@ -0,0 +1,155 @@
+
```

Comment:
Remove empty line

Reply:
Removed in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)